.gitlab-ci.yml

```diff
@@ -9,6 +9,7 @@ stages:
   - check
   - workspace
   - examples
+  - ink-waterfall
```

Comment:
adding a stage for just one job makes the rest of your pipeline (fuzz and publish) wait until this job's over. Are you sure you want to make it a blocker?